tests/L0/run_fused_layer_norm/test_fused_layer_norm.py

```diff
@@ -269,5 +269,41 @@ def test_autocast_fused_rms_norm(self, dtype, elementwise_affine):
 
 instantiate_device_type_tests(TestFusedLayerNorm, globals(), only_for=("cuda",))
 
+def _verify_export(fused, fused_x):
+    # check that export() is working
+    onnx_str = torch.onnx.export_to_pretty_string(fused, (fused_x,),
+                                                  input_names=['x_in'],
+    )
+    assert('x_in' in onnx_str)
+    assert('ReduceMean' in onnx_str)
```

Comment:
[nit-picking] [assert expression](https://docs.python.org/3/reference/simple_stmts.html?highlight=assert#the-assert-statement)

```suggestion
    assert 'x_in' in onnx_str
    assert 'ReduceMean' in onnx_str
```